projects/storefrontstyles/scss/layout/_index.scss

```diff
@@ -33,3 +33,7 @@ cx-page-layout {
 .LoginPageTemplate {
   @extend %LoginPageTemplate !optional;
 }
+
+.ErrorPageTemplate {
+  @extend %LoginPageTemplate !optional;
```

Comment:
Why .ErrorPageTemplate is extending %LoginPageTemplate?

Reply:
similar to the other https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/1249#discussion_r255499150:

> Yes, its a bit in between situation. Once all page templates are moved to this approach, i'm planning on eliminating backend templates. The front-end will probably lead, and AccountPageTemplate, LoginPageTemplate, NotFound, etc. are going to be merged. I need to come with a good name, and then it's time to move. I'm delaying this decision, first want to have good visibility on all of them.

Please bare with me. I'll do an update once we've passed all layouts.